demo/js/no-delay.js

```diff
@@ -0,0 +1,32 @@
+var { createFocusTrap } = require('../../dist/focus-trap');
+
+var container = document.getElementById('no-delay');
+
+var focusTrap = createFocusTrap(container, {
```

Comment:
This "no-delay" trap should be setting the `delayInitialFocus: false` option in its configuration.